src/protocol/handshakestate.c

```diff
@@ -1779,4 +1779,17 @@ int noise_handshakestate_get_handshake_hash
     return NOISE_ERROR_NONE;
 }
 
+
+size_t noise_handshakestate_get_size(const NoiseHandshakeState *state)
+{
+    return state->size;
+}
+
+
+NoiseDHState *noise_handshakestate_get_ephemeral_dh(NoiseHandshakeState *state)
+{
+    return state ? state->dh_local_ephemeral : 0;
```

Comment:
You should use NULL (which is generally defined as `(void*)0` (so a real pointer) instead of 0 to silence a potential conversion warning.